<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDock</class>
 <widget class="QDockWidget" name="PropertiesDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="m_search_properties_edit">
        <property name="placeholderText">
         <string>Search properties</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="flow::PropertiesBrowser" name="m_properties_browser"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_toolbar_layout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>flow::PropertiesBrowser</class>
   <extends>QTreeView</extends>
   <header>flow/document/component/properties/properties_browser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
